### ISSUE TITLE
[Accessibility] 9xMissing alternative text and 7xMissing form label - Register page

### ISSUE THREAD

**Issue by cmunteanuRW** [Talleh/VolleyballPub#3]
**Steps to reproduce:**

https://github.com/user-attachments/assets/44c9224a-c3ec-4473-bec8-8283b91379f6

**Actual Result:**

![Image](https://github.com/user-attachments/assets/9878fd18-741e-4c0d-940d-c766b22217db)

**Expected Result:**
No accessibility issues are present
